db/migrate/20150328164617_devise_create_users.rb

```diff
@@ -34,6 +34,8 @@ def change
       t.boolean :have_children
       t.string  :property_type
       t.string  :yard_type
+      t.string  :price
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.